src/auth/index.ts

```diff
@@ -0,0 +1,83 @@
+import express, { Request } from 'express'
+import cors from 'cors'
+import * as yup from 'yup'
+import * as jwt from 'jsonwebtoken'
+
+import { table, client } from '../index'
+
+export const router = express.Router()
+
+router.use(cors(
```

Comment:
@Alexandria Does this router need its own middleware added to it (the `cors` and `express.json()`) or is the middleware added to the whole server in `src/index.ts` serving the same purpose?

Reply:
Ah! You're right.  I removed the middleware from the auth route and tested and its working just fine.